app/build.gradle

```diff
@@ -17,14 +17,14 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
```

Comment:
this is a local file. But it hasn't yet been added to this project. Its available on my repo see [here](https://github.com/otormaigh/SMART/tree/master/app)